test/equatable_mixin_test.dart

```diff
@@ -664,4 +690,21 @@ void main() {
       expect(instanceA == instanceB, isFalse);
     });
   });
+
+  group('LegacyEqualityOverride', () {
```

Comment:
Just a note - this does not work great as a regression test because the failing test case would be the code not compiling (I suppose if you ran this test without the change in `lib`, the test will not even run).

It is good to have it for documentation and consistency nonetheless, I suppose.

Reply:
Yup, I just wanted to have something in place to prevent a regression 👍